packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -388,4 +388,11 @@ export class MetamaskPage implements WalletPage {
     await this.page.close();
     return address;
   }
+
+  async changeWalletAddress(addressNumber: number) {
+    await this.navigate();
+    await this.page.click('data-testid=account-menu-icon');
+    await this.page.click(`text=Account ${addressNumber}`);
```

Comment:
I guess you need pass full name of account. 
Because in a future its will be more flexible.

Reply:
done